force-app/main/default/lwc/calendarBody/calendarBody.js

```diff
@@ -1,27 +1,36 @@
-import { LightningElement, track } from 'lwc';
+import { LightningElement, track, wire } from 'lwc';
+import getObjects from '@salesforce/apex/CalendarWorkOrdersController.getObjects';
 
 export default class CalendarBody extends LightningElement {
     weekdays = ['mandag', 'tirsdag', 'onsdag', 'torsdag', 'fredag', 'lørdag', 'søndag'];
     reversedWeekdays = ['mandag', 'tirsdag', 'onsdag', 'torsdag', 'fredag', 'lørdag', 'søndag'].reverse();
-    calendarContainer = document.getElementsByClassName('calendar-container');
     monthNav = 0;
-    currentMonth = 'Test';
+    currentMonth = '';
+    monthDays = [];
+    eventsForMonth = [];
+
+    @track serviceAppointments;
+    @wire(getObjects, { record: 'ServiceAppointment', start: 'EarliestStartTime', endTime: 'DueDate' })
+    wiredGetAllServiceAppointments(result) {
```

Comment:
koden burde ikke anta hvilket objekt man bruker, så variablene burde hete "records", for eksempel

Reply:
Litt problemer med å få endret dette på linje 13-17, da vi sliter med å få tak i en generell startdato for alle typer objekter, men navnet er endret til serviceappointments
<img width="531" alt="Skjermbilde 2022-03-16 kl  12 13 39" src="https://user-images.githubusercontent.com/55350586/158577681-2cf8824f-b364-4de6-b338-f49304badf48.png">

Reply:
Eksempelvis her - les kommentaren som er skrevet og hvordan koden er løst på linje 90. 
<img width="985" alt="Skjermbilde 2022-03-16 kl  12 16 01" src="https://user-images.githubusercontent.com/55350586/158578077-e843eed6-301f-4b28-81be-a5faf0fb7f4f.png">

Reply:
Vi har fått ordnet API. den ligger i en ny branch som heter api-tryouts. Skal få merget den også.

Reply:
Vi har løst det med API. Nå fungerer koden med parametere. vi har testkode vi sender inn i calendar.js.